instrumentation/active_job/test/instrumentation/active_job/patches/active_job_callbacks_test.rb

```diff
@@ -205,20 +189,10 @@
       end
 
       it 'tracks correctly for jobs that do retry' do
-        ::ActiveJob::Base.queue_adapter = :async
-
-        RetryJob.perform_now
-        ::ActiveJob::Base.queue_adapter.shutdown
-
-        # 1 enqueue, 2 perform
-        _(spans.count).must_equal(3)
+        RetryJob.perform_later rescue nil
```

Comment:
Why `perform_later` instead of `perform_now`?

Reply:
`perform_now` executed the job in the main thread the first time it ran and then every subsequent run was being added to the queue being operated on by the thread pool. 

That use case did not make sense to me anymore now that I am testing it as if the job were enqueued and all instances were being picked up by a downstream worker.